File: src/main/java/net/frozenblock/configurableeverything/mod_compat/FrozenLibIntegration.java
package net.frozenblock.configurableeverything.mod_compat;

import net.frozenblock.configurableeverything.util.ConfigurableEverythingSharedConstantsKt;
import net.frozenblock.configurableeverything.util.ConfigurableEverythingUtilsKt;
import net.frozenblock.lib.integration.api.ModIntegration;

public class FrozenLibIntegration extends ModIntegration {
	public FrozenLibIntegration() {
		super("frozenlib");
	}

	@Override
	public void init() {
		ConfigurableEverythingUtilsKt.log("FrozenLib integration ran!", ConfigurableEverythingSharedConstantsKt.UNSTABLE_LOGGING);
	}
}

File: fabric/src/main/java/net/frozenblock/configurableeverything/mod_compat/FrozenLibIntegration.kt
package net.frozenblock.configurableeverything.mod_compat

import net.frozenblock.configurableeverything.util.UNSTABLE_LOGGING
import net.frozenblock.configurableeverything.util.log
import net.frozenblock.lib.integration.api.ModIntegration

internal object FrozenLibIntegration : ModIntegration("frozenlib") {
    override fun init() {
        log("FrozenLib integration ran!", UNSTABLE_LOGGING)
    }
}
